habitat_baselines/config/objectnav/ddppo_objectnav_hm3d.yaml

```diff
@@ -0,0 +1,74 @@
+BASE_TASK_CONFIG_PATH: "configs/tasks/objectnav_hm3d.yaml"
+CMD_TRAILING_OPTS: ["TASK_CONFIG.ENVIRONMENT.ITERATOR_OPTIONS.MAX_SCENE_REPEAT_STEPS", "50000"]
```

Comment:
Is there no better way to express this value than this field?  This is for overriding values from CLI isn't it? Could lead to unexpected behavior left in.

Reply:
I am not sure there is one. It was reused form the objectNav config.